pyproject.toml

```diff
@@ -58,18 +60,33 @@ filterwarnings = [
     "error",
 
     # Warnings raised by dependencies of dependencies, out of our control
+    # https://github.com/Cadene/pretrained-models.pytorch/issues/221
+    "ignore:.* is deprecated and will be removed in Pillow 10:DeprecationWarning:pretrainedmodels.datasets.utils",
     # https://github.com/pytorch/vision/pull/5898
+    "ignore:.* is deprecated and will be removed in Pillow 10:DeprecationWarning:torchvision.transforms.functional_pil",
     # https://github.com/rwightman/pytorch-image-models/pull/1256
-    "ignore:.*is deprecated and will be removed in Pillow 10:DeprecationWarning",
+    "ignore:.* is deprecated and will be removed in Pillow 10:DeprecationWarning:timm.data",
     # https://github.com/pytorch/pytorch/issues/72906
     # https://github.com/pytorch/pytorch/pull/69823
-    "ignore:distutils Version classes are deprecated. Use packaging.version instead.:DeprecationWarning",
+    "ignore:distutils Version classes are deprecated. Use packaging.version instead:DeprecationWarning:torch.utils.tensorboard",
     # https://github.com/PyTorchLightning/pytorch-lightning/issues/13256
-    "ignore:torch.distributed._sharded_tensor will be deprecated, use torch.distributed._shard.sharded_tensor instead:DeprecationWarning",
+    # https://github.com/PyTorchLightning/pytorch-lightning/pull/13261
+    "ignore:torch.distributed._sharded_tensor will be deprecated:DeprecationWarning:torch.distributed._sharded_tensor",
+    # https://github.com/rasterio/rasterio/issues/1742
+    # https://github.com/rasterio/rasterio/pull/1753
+    "ignore:Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated:DeprecationWarning:rasterio.crs",
+    # https://github.com/pytorch/pytorch/issues/60053
+    # https://github.com/pytorch/pytorch/pull/60059
+    "ignore:Named tensors and all their associated APIs are an experimental feature and subject to change.:UserWarning:torch.nn.functional",
+    # https://github.com/numpy/numpy/issues/14920
+    # https://github.com/numpy/numpy/issues/15748
+    "ignore:numpy.ufunc size changed, may indicate binary incompatibility:RuntimeWarning",
```

Comment:
Only occurs for the minimum version tests, but basically, at least one of our deps was built with a different version of numpy than we are using to run the tests. Doesn't seem to cause any harm other than the warning.